lib/membrane_caps_audio_raw.ex

```diff
@@ -283,6 +283,17 @@ defmodule Membrane.Caps.Audio.Raw do
     end
   end
 
+  @doc """
+  Returns a binary which corresponds to the silence during the given interval
+  of time in given caps' fromat
+  """
+  @spec sound_of_silence(Time.non_neg_t(), t, (float -> integer)) :: binary
+  def sound_of_silence(time, %__MODULE__{} = caps, round_f \\ &(&1 |> :math.ceil() |> trunc))
```

Comment:
I would change the arguments order to be consistent with `sound_od_silence/1` - caps should go first followed by the time and rounding function.